examples/other/sam2_point_segmentation.py

```diff
@@ -0,0 +1,256 @@
+# ruff: noqa: N802
+# sphinx_gallery_pygfx_docs = 'code'
+# sphinx_gallery_pygfx_test = 'off'
+
+"""
+Interactive Segmentation with SAM2
+==================================
+
+This script demonstrates a real-time interactive segmentation application using pygfx points as prompts to the SAM2 (Segment Anything Model 2) model and Qt (PySide6) for the GUI.
+
+Additional dependencies required to run this example:
+    pip install git+https://github.com/facebookresearch/sam2.git
```

Comment:
Thank you. 

Just a note I suggested that we use this strategy since the pypi package is not from the official distribution and quite old. 

Mostly the fact that it is highly not official and undocumented scares me.